orga/app/components/information-banner.js

```diff
@@ -0,0 +1,11 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+
+export default class InformationBanner extends Component {
+  @service currentUser;
+
+  get informationBannerMustBeDisplayed() {
+    return !this.currentUser.prescriber.areNewYearStudentsImported &&
+      this.currentUser.organization.type === 'SCO';
```

Comment:
Il faut aussi que l'orga puisse importer des élèves
```suggestion
     this.currentUser.isSCOManagingStudents;
```

Reply:
bien vu, merci je corrige